tests/unit/s2n_session_ticket_test.c

```diff
@@ -1097,9 +1118,13 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1), ticket_key1, s2n_array_len(ticket_key1), 0));
         EXPECT_SUCCESS(s2n_connection_set_config(server_conn, server_config));
 
-        /* Setup stuffers value containing the valid key name, valid iv and invalid encrypted blob */
+        /* Setup stuffers value containing the valid version number, valid key name, valid info, valid iv and invalid encrypted blob */
+        POSIX_GUARD(s2n_stuffer_write_uint8(&server_conn->client_ticket_to_decrypt, S2N_PRE_ENCRYPTED_STATE_V1));
```

Comment:
Have you now confirmed that these tests are failing for the right reason? And you've checked for other failure tests that you "broke"?

Reply:
Yeah there's basically two places we check that we fail to decrypt a ticket for a particular reason. One of them throws a very specific error, "key not found", so I'm certain that that test is working correctly. The other one throws a more generic "decrypt error", so I basically just stepped through the code to make sure the issue is the invalid encrypted blob, and not an invalid syntax.